src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -320,4 +323,11 @@ export class NgxSmartModalComponent implements OnInit, OnDestroy {
     this._renderer.setStyle(this.nsmContent.nativeElement, 'top', offsetTop + 'px');
     this._renderer.setStyle(this.nsmContent.nativeElement, 'left', offsetLeft + 'px');
   }
+
+  /**
+   * Is current platform browser
+   */
+  public isBrowser(): boolean {
```

Comment:
This method is only used in the component, so you can define it as private and make it a getter.

Reply:
Done.